cmd/delete/cluster.go

```diff
@@ -326,37 +296,44 @@ func NewClusterCmd() *cobra.Command {
 		"Location where to download schemas, defaults and the distribution manifests from. "+
 			"It can either be a local path (eg: /path/to/distribution) or "+
 			"a remote URL (eg: git::git@github.com:sighupio/distribution?depth=1&ref=BRANCH_NAME). "+
-			"Any format supported by hashicorp/go-getter can be used.",
+			"Any format supported by hashicorp/go-getter can be used",
 	)
 
 	clusterCmd.Flags().String(
 		"distro-patches",
 		"",
-		"Location where to download distribution's user-made patches from. "+
-			"Any format supported by hashicorp/go-getter can be used.",
+		"Location where the distribution's user-made patches can be downloaded from. "+
+			"This can be either a local path (eg: /path/to/distro-patches) or "+
+			"a remote URL (eg: git::git@github.com:your-org/distro-patches?depth=1&ref=BRANCH_NAME). "+
+			"Any format supported by hashicorp/go-getter can be used."+
+			" Patches within this location must be in a folder named after the distribution version (eg: v1.29.0) and "+
+			"must have the same structure as the distribution's repository",
 	)
 
 	clusterCmd.Flags().StringP(
 		"bin-path",
 		"b",
 		"",
-		"Path to the folder where all the dependencies' binaries are installed",
+		"Path to the folder where all the dependencies' binaries are downloaded",
 	)
 
 	clusterCmd.Flags().StringP(
 		"phase",
 		"p",
 		"",
-		"Limit execution to the specified phase. Options are: infrastructure, kubernetes, distribution",
+		"Limit execution to a specific phase. Options are: "+strings.Join([]string{
+			cluster.OperationPhaseInfrastructure,
+			cluster.OperationPhaseKubernetes,
+			cluster.OperationPhaseDistribution,
+		}, ", "),
 	)
 
 	if err := clusterCmd.RegisterFlagCompletionFunc("phase", func(_ *cobra.Command, _ []string, _ string) ([]string, cobra.ShellCompDirective) {
 		return []string{
-				cluster.OperationPhaseInfrastructure,
-				cluster.OperationPhaseKubernetes,
-				cluster.OperationPhaseDistribution,
-			},
-			cobra.ShellCompDirectiveDefault
+			cluster.OperationPhaseInfrastructure,
+			cluster.OperationPhaseKubernetes,
+			cluster.OperationPhaseDistribution,
+		}, cobra.ShellCompDirectiveDefault
```

Comment:
```suggestion
```

Reply:
Main phases includes `plugins` that is not currently supported by the `delete` command.